dev/artifacts/Products/Products.recipes

```diff
@@ -59,17 +58,17 @@ recipe
   GiftList
     person <- view1
   ProductMultiplexer
-    products <- view2
+    list <- view2
```

Comment:
I should have mentioned this, I'm not sure which is preferred (`products` or `list`) but I figured it was easier to use the preference from arcs.

`products` is more descriptive, though. Should I change this and the version in arcs to that?

Reply:
Fwiw, my hope is that soonish the Multplexers are generic, in which case the input parameter will need a generic name, like `list`.

Reply:
Oh yes! Thanks makes total sense. And there's the start to that in ProductMultiplexer2 (one of the parameters is generic).